app/pages/learn/01_tutorial/07_rich_client_apps/02_introduction-javafx-animation.md

```diff
@@ -0,0 +1,245 @@
+---
+id: javafx.animation
+title: Introduction to JavaFX animations
+slug: learn/javafx-animations
+type: tutorial
+group: rich-client-apps
+category: javafx
+category_order: 2
+layout: learn/tutorial.html
+subheader_select: tutorials
+main_css_id: learn
+description: "Learn to create advanced JavaFX animations"
+last_update: 2024-05-17
+---
+
+The [javafx.animation](javafxdoc:AnimationPackageSummary) package in JavaFX offers a simple framework for creating animations and transitions in a JavaFX application.
+It operates on the principle of [WritableValue\<T\>](javafxdoc:WritableValue) which are used across JavaFX.
```

Comment:
No, not really. I am kinda unsure where to place it or if it's a good idea to mention what they are in the beginning.

Reply:
As it seems to be quite fundamental for many other things, I think it might make sense to explain it before `Animation`. This way, it wouldn't get in between other concepts you are explaining.